### PR TITLE
Enhancement icd10 categories

### PR DESCRIPTION
**Pull request by DSWallach**
# Pull Request Template for MMEDS

#### Include each file that was changed, how it was changed (in broad terms), and why it was changed.
- sql/tables.sql: This has been updated to reflect the new scheme for storing ICD codes.
- mmeds/config.py: A few more globals are defined to reconcile the list of tables in the metadata with the list of tables in the databases. `(ICDCode -> (IllnessBroadCategory, IllnessCategory, IllnessDetails))`
- mmeds/database.py: Added a new method `load_ICD_codes`. It parses the `ICDCode` table into `IllnessBroadCategory`, `IllnessCategory`, and `IllnessDetails` tables and their necessary columns. It then runs the methods to create the import file for each table and load it into the database.
- mmeds/mmeds.py: A few functions were updated to ignore the `ICDCode` column/table when running checks. New functions are `load_ICD_codes` which creates a `dict` with all known ICD codes as keys and `check_ICD_codes` which checks the codes in the provided column against that dict.
- mmeds/test_files/User_Guide.txt: Updated to include instructions for ICD codes.
- mmeds/resources/icd10cm_codes_2018.txt: A file pulled from cms.gov containing all known ICD10 codes.

The Remaining changed files are tests and metadata files updated to work with the new database schema.

## Checklist of pre-requisites
- [x] Does the code run?  
- [x] Does the code follow the repository style?  
- [x] Is the code tested?  

#### Example: Functions A, B, and C are now all methods of Class D.

ICD 10 codes are split into three separate tables to allow for more complex queries.

### CODE REVIEW

**Comment by DSWallach** [comment]
@zachwilson Don't mind the CodeFactor failing. I have another branch where I fix the issues it finds. I haven't gotten the altered setup to run on travis yet though so for the time being I'll accept the bad grade.

File: mmeds/config.py [hunk 1]
```diff
@@ -284,6 +311,8 @@
 MIXS_MAP = {v: k for (k, v) in MMEDS_MAP.items()}
 
 
+
```
Comment (zachwilson) [inline mmeds/config.py]: What are these extra white space lines for?

Comment (DSWallach) [reply mmeds/config.py]: Getting the get_salt function out of the way. I removed them.

**Comment by DSWallach** [comment]
@DSWallach Addressed your commet

**Pull request merged**